Dockerfile

```diff
@@ -32,7 +32,9 @@ FROM docker.io/debian:bookworm-slim
 WORKDIR /opt/stalwart
 RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get update && \
-    apt-get install -yq ca-certificates
+    apt-get install -yq --no-install-recommends \
+      ca-certificates \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
```

Comment:
AFAIK the official debian docker images are already set up to run the equivalent of `apt-cache clean` so removing `/var/cache/apt` should not provide a significant benefit

Reply:
Please raise that at #1782